src/feature_extractor/feature_extractor_video.cc

```diff
@@ -15,25 +17,80 @@
 #include "mediapipe/framework/port/parse_text_proto.h"
 #include "mediapipe/framework/port/status.h"
 
+#include "src/calculators/array_to_csv_row_calculator.pb.h"
 #include "src/features/face/face_analyzer.h"
 #include "src/features/mouth/mouth_analyzer.h"
 #include "src/features/eye/eye_analyzer.h"
 
 // Extracts features from video stream.
 //
-// Build: 
-//     bazel build --define MEDIAPIPE_DISABLE_GPU=1 //src/feature_extractor:feature_extractor_video --nocheck_visibility
+// Build:
+//     bazel build --define MEDIAPIPE_DISABLE_GPU=1 \
+    //src/feature_extractor:feature_extractor_video --nocheck_visibility \
+    && bazel-bin/src/feature_extractor/feature_extractor_video \
+    --frame_width=640 --frame_height=480 --frame_rate=30
 // RUN:
 //     bazel-bin/src/feature_extractor/feature_extractor_video
 
+// Validates both the frame_width and frame_height
+// flags.
+static bool ValidateFrameSizeFlag(const char* flagname, int32 value) {
+    if (value > 0 && value < 20000) {
+        return true;
+    }
+
+    printf("Invalid value for --%s: %d\n", flagname, static_cast<int>(value));
+
+    return false;
+}
+
+static bool ValidateFrameRateFlag(const char* flagname, int32 value) {
+    if (value > 0 && value < 500) {
+        return true;
+    }
+
+    printf("Invalid value for --%s: %d\n", flagname, static_cast<int>(value));
+
+    return false;
+}
+
+DEFINE_int32(
+        frame_width, -1,
+        "Frame width for the input video/camera.");
+DEFINE_validator(frame_width, &ValidateFrameSizeFlag);
+
+DEFINE_int32(
+        frame_height, -1,
+        "Frame height for the input video/camera.");
+DEFINE_validator(frame_height, &ValidateFrameSizeFlag);
+
+DEFINE_int32(
+        frame_rate, -1,
+        "Frame rate for the input video/camera. " +
+            "Expressed in frames per second.");
+DEFINE_validator(frame_rate, &ValidateFrameRateFlag);
+
+DEFINE_string(
+        feature_file_path, "data/dataset/test.csv",
+        "Path to store the resulting file.");
+
 namespace mediapipe {
 
 mediapipe::Status RunVideoReader() {
-
+    // TODO(@jeanchilger):
+    //      - Removes a window of N seconds from beggining
     CalculatorGraphConfig config = ParseTextProtoOrDie<CalculatorGraphConfig>(R"(
         input_stream: "in"
-        output_stream: "out_landmarks"
+        input_stream: "video_width"
+        input_stream: "video_height"
+        input_stream: "video_fps"
         output_stream: "out_image"
+        output_stream: "out_vector"
+        profiler_config {
+            trace_enabled: true
+            enable_profiler: false
+            trace_log_interval_count: 100
```

Comment:
I have a question regarding this parameters, what they do during running time

Reply:
Oh, I forgot that :(, it was used for "debugging". Basically this tells mediapipe to save a "trace file", which can be used to import the graph to the [visualizer](https://viz.mediapipe.dev/) and check inputs, and things like that. More information on [this link](https://google.github.io/mediapipe/tools/tracing_and_profiling.html).